### PR TITLE
randomFloat to faker methods

### PR DESCRIPTION
**Pull request by josimarlemos**
Issue [https://github.com/drawmyattention/laravel-make-resource/issues/1](https://github.com/drawmyattention/laravel-make-resource/issues/1)

### CODE REVIEW

**Comment by drawmyattention** [comment]
Can you confirm this fixes #1 ?

**Comment by josimar-lemos** [comment]
Yes.

File: tests/ResourceGeneratorTest.php [hunk 1]
```diff
@@ -152,6 +152,27 @@ public function it_accepts_attributes_which_are_saved_to_the_model()
...
 
     /** @test */
+    public function it_accepts_float_attributes_which_are_saved_to_the_model()
+    {
+        $model = 'Louse';
+
+        // When I create a new model called 'Animal'.
+        $this->runArtisanCommand(ResourceMakeCommand::class, [
+            'name' => $model,
+            'attributes' => 'eyes:float',
```
Comment (drawmyattention) [inline tests/ResourceGeneratorTest.php]: The example attribute of `eyes` doesn't really fit with a float data type. Can you change this to `weight`, or something that could reasonably be a float value?

Comment (josimar-lemos) [inline tests/ResourceGeneratorTest.php]: Sorry for the easter egg
"which animal has more than three and less than four eyes"
It is **PI**olho
**PI**olho is a Louse and eyes is a "olho" in portuguese.
And PI equals 3.14....

**Pull request closed without merging**